ReactiveObjCBridge/RACScheduler+SwiftSupport.h

```diff
@@ -0,0 +1,5 @@
+#import <ReactiveObjC/ReactiveObjC.h>
+
+@interface RACScheduler (SwiftSupport)
++ (RACScheduler *) schedulerWithRACSwiftScheduler:(RACScheduler*)scheduler;
```

Comment:
This is needed to circumvent the initialisation model in Swift, which can't express factories of class clusters just yet.